ReportUtilityBills/measure.rb

```diff
@@ -0,0 +1,792 @@
+    arg.setDisplayName('Output Format')
+    arg.setDescription('The file format of the annual (and timeseries, if requested) outputs.')
+    arg.setDefaultValue('csv')
+    args << arg
+
+    # electricity_bill_type_choices = OpenStudio::StringVector.new
+    # electricity_bill_type_choices << 'Simple'
+    # electricity_bill_type_choices << 'Detailed'
+
+    # arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('electricity_bill_type', electricity_bill_type_choices, true)
+    # arg.setDisplayName('Electricity: Simple or Detailed')
+    # arg.setDescription("Choose either 'Simple' or 'Detailed'. If 'Simple' is selected, electric utility bills are calculated based on user-defined fixed charge and marginal rate. If 'Detailed' is selected, electric utility bills are calculated based on either: a tariff from the OpenEI Utility Rate Database (URDB), or a real-time pricing rate.")
+    # arg.setDefaultValue('Simple')
+    # args << arg
+
+    # utility_rate_type_choices = OpenStudio::StringVector.new
+    # utility_rate_type_choices << 'Autoselect OpenEI'
+    # utility_rate_type_choices << 'Sample Real-Time Pricing Rate'
+    # utility_rate_type_choices << 'Sample Tiered Rate'
+    # utility_rate_type_choices << 'Sample Time-of-Use Rate'
+    # utility_rate_type_choices << 'Sample Tiered Time-of-Use Rate'
+    # utility_rate_type_choices << 'User-Specified'
+
+    # arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('electricity_utility_rate_type', utility_rate_type_choices, false)
+    # arg.setDisplayName('Electricity: Utility Rate Type')
+    # arg.setDescription("Type of the utility rate. Required if electricity bill type is 'Detailed'.")
+    # arg.setDefaultValue('Autoselect OpenEI')
+    # args << arg
+
+    # arg = OpenStudio::Measure::OSArgument::makeStringArgument('electricity_utility_rate_user_specified', false)
+    # arg.setDisplayName('Electricity: User-Specified Utility Rate')
+    # arg.setDescription("Absolute/relative path of the json. Relative paths are relative to the HPXML file. Required if utility rate type is 'User-Specified'.")
+    # args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electricity_fixed_charge', false)
+    arg.setDisplayName('Electricity: Fixed Charge')
+    arg.setUnits('$/month')
+    arg.setDescription('Monthly fixed charge for electricity.')
+    arg.setDefaultValue(12.0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('electricity_marginal_rate', false)
+    arg.setDisplayName('Electricity: Marginal Rate')
+    arg.setUnits('$/kWh')
+    arg.setDescription('Price per kilowatt-hour for electricity.')
+    arg.setDefaultValue(Constants.Auto)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('natural_gas_fixed_charge', false)
+    arg.setDisplayName('Natural Gas: Fixed Charge')
+    arg.setUnits('$/month')
+    arg.setDescription('Monthly fixed charge for natural gas.')
+    arg.setDefaultValue(8.0)
```

Comment:
I suggest we increase this to 12.0 just like electricity. [This source](https://www.aga.org/sites/default/files/aga_energy_analysis_-_natural_gas_utility_rate_structure.pdf) suggests the average was $11.25 in 2015.